driver/csiplugin/controllerserver.go

```diff
@@ -1515,7 +1517,7 @@ func (cs *ScaleControllerServer) DeleteVolume(ctx context.Context, req *csi.Dele
 
 	volumeIdMembers, err := getVolIDMembers(volumeID)
 	if err != nil {
-		return &csi.DeleteVolumeResponse{}, err
+		return &csi.DeleteVolumeResponse{}, nil
```

Comment:
@hemalathagajendran what is the testcase failing for this.

Reply:
Test case : should succeed when an invalid volume id is used during "deleteVolume" operation